Scripts/build.swift

```diff
@@ -1,178 +0,0 @@
-#!/usr/bin/env swift
-
-import Foundation
-
-// Usage: build.swift <spm|xcode> <platform> [<path_to_xcpretty>]
-
-func execute(commandPath: String, arguments: [String], pipedTo pipeProcess: Process? = nil) throws {
-	let task = Process()
-	task.launchPath = commandPath
-	task.arguments = arguments
-
-	let argumentsString = arguments
-		.map { argument in
-			if argument.contains(" ") {
-				return "\"\(argument)\""
-			} else {
-				return argument
-			}
-		}
-		.joined(separator: " ")
-
-	if let pipeProcess = pipeProcess, let pipePath = pipeProcess.launchPath {
-		let pipe = Pipe()
-		task.standardOutput = pipe
-		pipeProcess.standardInput = pipe
-
-		print("Launching command: \(commandPath) \(argumentsString) | \(pipePath)")
-
-	} else {
-		print("Launching command: \(commandPath) \(argumentsString)")
-	}
-
-	task.launch()
-
-	pipeProcess?.launch()
-
-	task.waitUntilExit()
-
-	guard task.terminationStatus == 0 else {
-		throw TaskError.code(task.terminationStatus)
-	}
-}
-
-enum TaskError: Error {
-	case code(Int32)
-}
-
-enum Platform: String, CustomStringConvertible {
-	case iOS_13
-
-	var destination: String {
-		switch self {
-		case .iOS_13:
-			return "platform=iOS Simulator,OS=13.7,name=iPhone 11 Pro"
-		}
-	}
-
-	var derivedDataPath: String {
-		return ".build/derivedData/\(rawValue)"
-	}
-
-	var description: String {
-		return rawValue
-	}
-}
-
-enum Task: String, CustomStringConvertible {
-	case spm
-	case xcode
-
-	var workspace: String? {
-		switch self {
-		case .xcode:
-			return "Example/Stagehand.xcworkspace"
-		case .spm:
-			return nil
-		}
-	}
-
-	var project: String? {
-		switch self {
-		case .spm:
-			return "generated/Stagehand.xcodeproj"
-		case .xcode:
-			return nil
-		}
-	}
-
-	var scheme: String {
-		switch self {
-		case .xcode:
-			return "Stagehand Demo App"
```

Comment:
@luispadron I appreciate your trying to update CI here, but unless I'm missing something we've made a bit of an oops.

In this PR we've removed the instruction to run the demo app tests, and the demo app is where the tests for this project live.

You mention in the PR description that:
> the `pod lib lint` job still exists and continues to validate the podspec which is what we care about

However, [the podspec](https://github.com/cashapp/stagehand/blob/master/Stagehand.podspec) does **not** list any tests, and therefore I believe this PR actually removed any validation that Stagehand is behaving correctly. All we're testing in CI now is that Stagehand builds, which is pretty insufficient.

Moreover, by no longer building the sample app, it is now possible for the sample code shipped with Stagehand to not compile, which is less-than-ideal.